src/fides/api/ops/api/v1/scope_registry.py

```diff
@@ -228,3 +229,8 @@
 }
 
 SCOPE_REGISTRY = list(SCOPE_DOCS.keys())
+
+
+ScopeRegistryEnum = Enum(  # type: ignore[misc]
+    "ScopeRegistryEnum", {scope: scope for scope in SCOPE_REGISTRY}  # type: ignore
```

Comment:
super nit, but the `RoleRegistry` enum is just called `RoleRegistry` while this one is `ScopeRegistryEnum`. might be good to make them consistent (at least for me consuming these, would make things a bit clearer 😄 )

```suggestion
    "ScopeRegistry", {scope: scope for scope in SCOPE_REGISTRY}  # type: ignore
```

Reply:
good call!